docs/api_docs/bundle.yaml

```diff
@@ -346,6 +359,27 @@ definitions:
       body_from_file:
         type: string
         description: file name (relative to working directory of OM) to load HTTP body from
+  ActionReplyGRPC:
+    type: object
+    description: reply to incoming GRPC that triggered this behavior with a response
+    properties:
+      headers:
```

Comment:
I added headers here just in case if people may want to override grpc status code or others in the headers.